sdk/servicebus/Azure.Messaging.ServiceBus/tests/Administration/ServiceBusManagementClientLiveTests.cs

```diff
@@ -29,6 +31,21 @@ public class ServiceBusManagementClientLiveTests : RecordedTestBase<ServiceBusTe
             _serviceVersion = serviceVersion;
         }
 
+        [SetUp]
+        public void AddSanitizers()
+        {
+            Recording.AddBodySanitizer(
```

Comment:
File an issue to add XPath sanitizer support

Reply:
https://github.com/Azure/azure-sdk-tools/issues/2334